src/deprecated/block-defaults/custom-block-styles.php

```
@@ -174,11 +420,19 @@ public static function validate_boolean( $value, $request, $param ) {
-			$slug = $request->get_param( 'slug' );
-			$name = $request->get_param( 'name' );
-			$data = $request->get_param( 'data' );
-			$save = $request->get_param( 'save' );
+			$block = $this->sanitize_block_name( $request->get_param( 'block' ) );
+			$slug = $this->sanitize_style_slug( $request->get_param( 'slug' ) );
+			$name = sanitize_text_field( $request->get_param( 'name' ) );
+			$data = $this->sanitize_block_style_data( $request->get_param( 'data' ) );
+			$save = $this->sanitize_block_style_save( $request->get_param( 'save' ) );
+
+			if ( empty( $block ) || empty( $slug ) ) {
+				return new WP_Error(
```

PHPStan (2.2.2)

[error] Method Stackable_Custom_Block_Styles::update_block_style() with return type void returns WP_Error but should not return anything.


(return.void)